INSTALLING_CLI_TOOLS.md

```diff
@@ -1,3 +1,14 @@
+
+## The new BF CLI replaces legacy standalone tools
+
+The Bot Framework SDK team is happy to announce the General Availability of the consolidated bot framework CLI tool [bf-cli](https://aka.ms/bfcli). The new BF CLI tool will replace legacy standalone tools to manage Bot Framework bots and related services. The old tools will be ported over in phases and all new features, bug fixes, and further investments will focus on the new BF CLI.  Old tools will still work for the time being, but they are going to be deprecated in future releases.
+
+Upon the release of Bot Framework SDK version 4.6 the following legacy tools have been ported: Chatdown, QnAMaker, LuisGen, and LuDown.
+
+To learn more about the BF CLI please visit the following github repository: https://aka.ms/bfcli
```

Comment:
```suggestion
To learn more about the BF CLI please visit the [BF CLI github repository](https://aka.ms/bfcli).
```

Reply:
naked aka.ms link is weird since you said it was a github repo. If you want to keep the aka.ms link, just hide it in a link.